src/js/constants/ServiceConfig.js

```diff
@@ -0,0 +1,18 @@
+let ServiceConfig = {
+  BLACK_LIST: [
+    'uris',
+    'ports',
+    'version',
+    'versionInfo',
+    'deployments',
+    'tasks',
+    'taskStats',
+    'tasksHealthy',
+    'tasksRunning',
+    'tasksStaged',
+    'tasksUnhealthy',
+    '_itemData'
```

Comment:
This should still not be the case. This means that we are instantiating a struct with struct data, and not raw data